staking/contracts/contracts.py

```diff
@@ -12,13 +14,13 @@ def is_admin() -> Expr:
     return Assert(Txn.sender() == App.globalGet(Bytes("A")))
 
 @Subroutine(TealType.none)
-def set_admin(addr: Expr):
+def set_admin(addr: Expr) -> Expr:
```

Comment:
On line 19 (and throughout) we see usages of string literals representing local and global states. These ought to be constants. e.g. instead of calling `Bytes("A")` whenever access to the admin's address is desired, there could be a constant at to top of this file

```python
ACCOUNT_ADDRESS_GLOBAL_STATE = Bytes("A")
```